druid/src/theme.rs

```diff
@@ -50,6 +50,8 @@ pub const SCROLL_BAR_PAD: Key<f64> = Key::new("scroll_bar_pad");
 pub const SCROLL_BAR_RADIUS: Key<f64> = Key::new("scroll_bar_radius");
 pub const SCROLL_BAR_EDGE_WIDTH: Key<f64> = Key::new("scroll_bar_edge_width");
 
+pub const DEFAULT_ANIMATION_DURATION: Key<u64> = Key::new("default_animation_duration");
```

Comment:
Not sure about this one. It might make sense to have reasonable defaults for animation duration. Until this PR, I did not notice any time-based animation implementations.